src/main/java/seedu/organizer/logic/commands/ToggleCommand.java

```diff
@@ -19,7 +23,7 @@
 import seedu.organizer.model.task.exceptions.TaskNotFoundException;
 
 /**
- * Inverse the value of task status
+ * Inverse the value of task status (Done or Not done)
```

Comment:
Perhaps this comment should also clarify if subtasks are also considered done or not based on the main task's completion? =P